### PR TITLE
Update app bar tooltip text to `Toggle Linked Boards`

### LINKED ISSUE #3659: Feature Idea: improve the RHS App Bar Boards tooltip text
## Summary

Improve the RHS App Bar Boards tooltip text:

image

cc @justinegeffen perhaps `Toggle Linked Boards`?

## How important this is to me and why

Importance: High/Medium/Low

Use cases:
1.
2.
3.

## Additional context/similar features

I'm partial to Playbooks' usage of `Toggle`.

### PR DESCRIPTION
**Pull request by Pinjasaur**
#### Summary

image

#### Ticket Link

closes #3659

### CODE REVIEW

**Comment by Pinjasaur** [comment]
cc @lindy65, not sure if you're using the app bar in the Rainforest test servers

File: mattermost-plugin/webapp/src/index.tsx [hunk 1]
```diff
@@ -311,7 +311,7 @@ export default class Plugin {
 
             if (this.registry.registerAppBarComponent) {
                 const appBarIconURL = windowAny.baseURL + '/public/app-bar-icon.png'
-                this.registry.registerAppBarComponent(appBarIconURL, () => mmStore.dispatch(toggleRHSPlugin), 'Boards')
+                this.registry.registerAppBarComponent(appBarIconURL, () => mmStore.dispatch(toggleRHSPlugin), 'Toggle Linked Boards')
```
Comment (sbishel) [inline mattermost-plugin/webapp/src/index.tsx:314]: This should be translated. Yes, it was an issue before as well.

**Review by sbishel — CHANGES_REQUESTED** [review]

**Comment by Pinjasaur** [comment]
@sbishel done—tried modelling it after Playbooks' i.e. `<FormattedMessage>` (https://github.com/mattermost/mattermost-plugin-playbooks/blob/d0d704cb446a2bdd075ab61e943465a417c93ba9/webapp/src/components/channel_header.tsx#L40) but that wasn't working as-is (seemed to need a ancestor `<IntlProvider>` component) so I tried `createIntl` using the i18n logic we already have within Focalboard as a product

**Comment by lindy65** [comment]
Thanks for the heads-up @Pinjasaur :) Currently, we're not using the apps bar in Rainforest.

File: webapp/i18n/en.json [hunk 2]
```diff
@@ -136,10 +137,20 @@
   "EditableDayPicker.today": "Today",
   "Error.mobileweb": "Mobile web support is currently in early beta. Not all functionality may be present.",
   "Error.websocket-closed": "Websocket connection closed, connection interrupted. If this persists, check your server or web proxy configuration.",
+  "Filter.contains": "contains",
```
Comment (wiggin77) [inline webapp/i18n/en.json:140]: Wow.  It looks like 118n extraction hadn't been run in a while.  We should add that to our CI/lint.

Comment (wiggin77) [reply webapp/i18n/en.json:140]: I'll create a ticket for that.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by sbishel — APPROVED** [review]
@Pinjasaur - Lots of work for such a minor change. :)

**Pull request merged**